File: Web/RMA-Web-Shell/projects/fincare/src/app/shared/models/allocated-payment.ts

export class AllocatedPayment {
  debtorPaymentId: number;
  bankStatementEntryId: number;
  debtorName: string;
  invoiceNumber: string;
  policyNumber: string;
  userReference: string;
  transactionDate: Date;
  statementDate: Date;
  hyphenDateProcessed: Date;
  hyphenDateReceived: Date;
  amount: number;
  bankAccountNumber: number;
  userReference1: string;
  userReference2: string;
  transactionType: string;
  schemeName: string;
  brokerName: string;
  policyStatus: string;
  clientType: string;
  allocationDate: Date;
  isExpanded?: boolean;
  debtorNumber: string;
  productCategory: string;
}

File: Service/Billing/RMA.Service.Billing.Contracts/Entities/AllocatedPayment.cs
using System;

namespace RMA.Service.Billing.Contracts.Entities
{
    public class AllocatedPayment
    {
        public int DebtorPaymentId { get; set; }
        public int BankStatementEntryId { get; set; }
        public string UserReference { get; set; }
        public string DebtorName { get; set; }
        public string InvoiceNumber { get; set; }
        public string PolicyNumber { get; set; }
        public DateTime? TransactionDate { get; set; }
        public DateTime? StatementDate { get; set; }
        public DateTime? HyphenDateProcessed { get; set; }
        public DateTime? HyphenDateReceived { get; set; }
        public decimal Amount { get; set; }
        public Int64 BankAccountNumber { get; set; }
        public string UserReference1 { get; set; }
        public string UserReference2 { get; set; }
        public string TransactionType { get; set; }
        public string SchemeName { get; set; }
        public string BrokerName { get; set; }
        public string PolicyStatus { get; set; }
        public string ClientType { get; set; }
        public DateTime? AllocationDate { get; set; }
        public string DebtorNumber { get; set; }
        public string ProductCategory { get; set; }
    }
}
